packages/builder/src/components/portal/licensing/DayPassWarningModal.svelte

```diff
@@ -12,7 +12,10 @@
 
   $: daysRemaining = $licensing.quotaResetDaysRemaining
   $: quotaResetDate = $licensing.quotaResetDate
-  $: dayPassesUsed = $licensing.usageMetrics?.dayPasses
+  $: dayPassesUsed =
+    $licensing.usageMetrics?.dayPasses > 100
+      ? 100
+      : $licensing.usageMetrics?.dayPasses
```

Comment:
NAB: Could go for a cheeky `Math.min($licensing.usageMetrics?.dayPasses, 100)` here!